README.md

```diff
@@ -1,28 +1,43 @@
-![Screenshot of the main page](./screenshot.jpg)
+![Screenshot of the application](./screenshot.jpg)
 
 # Unified Auctions UI
 
-The tool to enable easy and straightforward participation in [MakerDAO](https://makerdao.com/) liquidations for users both completely unfamiliar with Maker protocol but also experts.
+The tool to enable easy and straightforward participation in [MakerDAO](https://makerdao.com/) liquidations for users both completely unfamiliar with Maker protocol but also experts. The repository contains both the `frontend`-only application to fetch and manually interact with the maker auctions, as well as the `bot` – the "keeper" application to configure and run with your own settings to automatically particiate in specified auctions.
+
+### Using the app
+
+The frontend app is officially deployed on the [unified-auctions.makerdao.com](https://unified-auctions.makerdao.com/)
+
+### Running locally
+
+Besides being [officially deployed](https://unified-auctions.makerdao.com/) on the public internet, the app can also be run locally in case there are problems with the website avialablility. In order to simplify this process, we packaged the frontend application into the crossplatform electron app.
+
+1. Download relevant application from the [latest github release](https://github.com/sidestream-tech/unified-auctions-ui/releases/latest) page
+    - MacOS: download the file ending on `.dmg`
```

Comment:
Is the `.dmg` file enough or do we also need to refer to the `dmg.blockmap` (same for windows)